dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/authentication/ResetPasswordResourceIntegrationTest.java

```diff
@@ -70,12 +72,14 @@ public void testNoSuchUserException() throws DotSecurityException, NoSuchUserExc
     }
 
     @Test
-    public void testTokenInvalidException() throws DotSecurityException, NoSuchUserException, DotInvalidTokenException {
+    public void testTokenInvalidException() throws DotSecurityException, NoSuchUserException, DotInvalidTokenException, DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)